### PR TITLE
Add instruction on how to use custom domain

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: custom_dc/setup_gcp.md [hunk 1]
```diff
@@ -8,15 +8,10 @@ published: true
 
 ## Overview
 
-Custom Data Commons is deployed in Google Cloud Platform (GCP). The follow
-manual presents how to install a custom Data Commons instance in an existing GCP
+Custom Data Commons is deployed in Google Cloud Platform (GCP). This manual
+presents how to install a custom Data Commons instance in an existing GCP
```
Comment (pradh) [inline custom_dc/setup_gcp.md:12]: nit: s/presents/describes/

File: custom_dc/setup_gcp.md [hunk 2]
```diff
@@ -37,11 +32,22 @@ The installation can use an existing domain, or create a default domain
...
 
+1. [Optional] The default domain of the instance is
+   `<PROJECT_ID>-datacommons.com`. If you want to use an existing custom domain,
+   set the environment variable:
+
+   ```bash
+   export CUSTOM_DC_DOMAIN=<existing_domain>
+   ```
+
+   Later on, need to create a new DNS record by linking the domain with the IP
```
Comment (pradh) [inline custom_dc/setup_gcp.md:43]: nit: "you need to create..."

so its more obvious its on them?

**Review by pradh — APPROVED** [review]

File: custom_dc/setup_gcp.md (hunk 1 above)
Comment (shifucun) [reply custom_dc/setup_gcp.md]: Done

File: custom_dc/setup_gcp.md (hunk 2 above)
Comment (shifucun) [reply custom_dc/setup_gcp.md]: Done

**Review by shifucun — COMMENTED** [review]
thanks!

**Pull request merged**